### DISCUSSION TITLE
How to change output directory?

### DISCUSSION THREAD

**Discussion by GGNode** [idaholab/moose#22986] [Q&A General] [answered]
When I ran a Moose parallel computation on a HPC cluster, I found that the computation speed was not as fast as I thought. I think I should change the output directory because the program was trying to write input_out.e to a virtual node which was restricted by internet speed. But I cannot find any useful tips for me to **change the output directory** **where I can place the input_out.e file**. Can anyone give me some suggestions?

**Comment by maxnezdyur** [comment, marked as answer]
In the output block, you can specify the location where you want to save something using `file_base`. For example 
`file_base = /home/location_where_i_want_my_file/name_i_want_to_give_my_file`

**Reply by achourasia** [reply]
Is there a way to specify only the output directory instead of full path to the output file? The idea is to keep the default output name by only specifying the output directory.

**Reply by GiudGiud** [reply]
Hello

Not that I can see in the object parameters but feel free to contribute the feature, we would accept that change

Guillaume

**Reply by Yulongggggg** [reply]
Hi Guid do we have this feature now?
I really want to specify only the output directory instead of full path to the output file

**Reply by GiudGiud** [reply]
Hello

No we have not, we still use the file_base for that purpose

**Comment by GGNode** [comment]
Thank you very much!
